cloud-azure/src/main/resources/templates/arm-v2.ftl

```diff
@@ -544,7 +544,7 @@
                     "name": "${loadBalancer.name}-publicIp",
                     "location": "[parameters('region')]",
                     "sku": {
-                        <#if (loadBalancer.instanceGroupNames?size > 1)>
+                        <#if (loadBalancers?filter(loadBalancer -> loadBalancer.instanceGroupNames?size > 1)?size > 0)>
```

Comment:
If I'm reading this correctly, this is saying that if we have a load balancer that is routing traffic to more than one instance group, use the Standard SKU. Otherwise use the Basic SKU.

I think longer term we're going to have to figure out how to get the Standard SKU working. I can't think of a use case right now where this would be a problem, but now that the load balancer logic is there I've seen a few potential uses for it in other efforts come up. So really just flagging this for myself to keep it in mind.